.github/actions/test/action.yml

```diff
@@ -5,6 +5,11 @@ inputs:
   TARGET:
     required: true
 
+  SUDO:
+    description: 'Set to "sudo" to run the build with sudo, or leave undefined to run as the current user'
```

Comment:
It seems that the description should be:

> Set it to an empty string to run the tests as the current user, leave it with the default value to test with "sudo"

Reply:
Wait, it seems that GHA will treat an empty string as the default value per the [CI log](https://github.com/nix-rust/nix/actions/runs/7775039226/job/21200535550?pr=2269)?